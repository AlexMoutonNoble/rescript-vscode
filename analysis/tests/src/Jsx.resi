@react.component
let make: (~first: string) => React.element
//  ^hov

let y : int
//  ^hov