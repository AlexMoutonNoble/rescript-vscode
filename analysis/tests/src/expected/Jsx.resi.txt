Hover tests/src/Jsx.resi 1:4
null

Hover tests/src/Jsx.resi 4:4
{"contents": "```rescript\nint\n```"}

